services/distribution/src/main/java/app/coronawarn/server/services/distribution/statistics/file/MockStatisticJsonFileLoader.java

```diff
@@ -10,7 +11,7 @@
 
 @Component
 @Profile("local-json-stats")
-public class LocalStatisticJsonFileLoader implements JsonFileLoader {
+public class MockStatisticJsonFileLoader implements StatisticJsonFileLoader {
```

Comment:
what is it 'mocking'? why we do have 'mocks' in productive code?

Reply:
Hi Hilmar. This is the statistics json file loader for local machine. It was already there when I started implementing. I renamed it from LocalStatistics to MockStatistics in order to avoid confusion because now we have all this classes starting with LocalStatistics which in fact mean "Regional" or "Federal state grouped". This class used the LocalStatistic naming to refer to "Local" environment.